### ISSUE TITLE
Consistent bracketing style

### ISSUE THREAD

**Issue by farziengineer** [farziengineer/underscore_cpp#78]
Some places have opening brackets starting on same line, while some of them have brackets on a new line.

**Comment by farziengineer** [comment]
Better to go with brackets on new line

**Comment by gubatron** [comment]
yes, more C++ style. (it's hard for me to not do this unconsciously, been coding in java daily for 14 years), I think I'm going to add a hook before commits that applies proper clang-format to it and forbid's me from comitting if I have a mistake.
